example-get-started/code/.github/workflows/cml.yaml

```diff
@@ -27,6 +27,7 @@ jobs:
             git fetch origin main:main
           fi
 
+          dvc pull eval
```

Comment:
I'm not sure why we need this now (seems a regression, w/o it metrics are not shown in the `dvc exp diff` below) cc @dberenbaum . Also, I'm not sure I understand why this command is enough to get both revisions.

Reply:
Are you sure dvc-tracked metrics were ever pulled automatically? Up until now, we so often assumed metrics were git-tracked that I'm not sure we ever paid enough attention to this.

Reply:
I guess we need to fix this if we plan to start recommending dvc-tracked metrics

Reply:
Let's open an issue after this is deployed since it should be easy to reproduce from the deployed repo

Reply:
no, I'm not sure at all. Agreed that we need to fix this. Becomes a priority I guess. Since we always though had to cache them I would not be surprised if at some point we were fetching them.

Reply:
One more thought: haven't had a chance to test, but it might also work with stage-level metrics but not top-level metrics.

Reply:
@skshetry WDYT about adding this to https://github.com/iterative/dvc/issues/9722?